### DISCUSSION TITLE
Hey team wanted to say I’m very [Jun 04]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5866] [Discord Threads]
> 🙋‍♂️ **fromq1234**

**Comment by o1lab** [comment]
> 👨‍💻 **pranavxc**  
#### You can use our sdk and upload the file, for sample refer - https://github.com/nocodb/nocodb/blob/develop/packages/nc-gui/components/cell/attachment/utils.ts#L183   
####    
#### Or you can refer - https://github.com/nocodb/nocodb/discussions/369 and https://all-apis.nocodb.com/#tag/Storage/operation/storage-upload  
#  
> 🙋‍♂️ **fromq1234**  
#### Like I said it’s sparse and out of date.  The official api has no examples and is just wrong. The GitHub discussion is correct but the endpoint is from an older version (“/dashboards”). While I’m sure the SDK is great for typescript devs, it’s not exactly easy to consume for python devs  
#  
> 🙋‍♂️ **fromq1234**  
#### I spent hours looking over those two links and figured this out myself  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1103899004229652510)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
